packages/rocketchat-slackbridge/slackbridge.js

```diff
@@ -1018,6 +1025,100 @@ class SlackBridge {
 			}
 		}
 	}
+
+	/*
+	 https://api.slack.com/events/message/message_deleted
+	 */
+	processSlackMessageDeleted(slackMessage) {
+		if (slackMessage.previous_message) {
+			let rocketChannel = this.getRocketChannel(slackMessage);
+			let rocketUser = RocketChat.models.Users.findOneById('rocket.cat', { fields: { username: 1 } });
```

Comment:
It's returning the rocket.cat's `username` cuz you can change it, but the user's id will be always `rocket.cat`